spec/e2e-video.spec.js

```diff
@@ -1,9 +1,12 @@
 const path = require('path');
 const fakeDialog = require('spectron-fake-dialog');
 const helper = require('./helpers/spectron_helper');
+const rimraf = require('rimraf');
+const fs = require('fs');
 
 describe('E2E - Video', () => {
     beforeAll((done) => {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
```

Comment:
Is this needed still after you got it working?

Reply:
This was more of an issue of it taking varying times to finish writing the files